_posts/2019-06-3-4-contribute-open-source.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Contributing to open source"
```

Comment:
@bronzdoc Try titleizing this title. Follow the capitalization we have been using in past blog articles.